app/test_main.py

```diff
@@ -1,3 +1,73 @@
+from typing import Any
+
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
```

Comment:
The test case `(0, 0, [0, 0])` seems to be correct if the logic is that animals with 0 years should result in 0 human years. However, ensure that the logic in `get_human_age` supports this expectation.